python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -365,6 +364,31 @@ def run(self, science, matchedTemplate, difference,
                                    positiveFootprints=positives,
                                    negativeFootprints=negatives)
 
+    def _prepareInputs(self, difference):
+        """Ensure that we start with an empty detection and deblended mask.
+
+        Parameters
+        ----------
+        difference : `lsst.afw.image.ExposureF`
+            The difference image that will be used for detecting diaSources.
+            The mask plane will be modified in place.
+
+        Raises
+        ------
+        pipeBase.UpstreamFailureNoWorkFound
```

Comment:
I think this has to be like this in docs?

```suggestion
        lsst.pipe.base.UpstreamFailureNoWorkFound
```